contrib/enfuseAdvanced.lua

```diff
@@ -45,7 +45,7 @@ If you have a specific set of parameters you frequently like to use, you can sav
 
 target file:
 Select your file destination path, or check the "save to source image location" option.
-Unless "Create unique filename" is check, it will overwrite existing files
+Unless "Create unique filename" is selected, it will overwrite existing files
```

Comment:
Should the default be non destructive, i.e. create unique filename, just in case the user isn't paying attention?  That way the user would have to intentionally select the destructive behavior.

Reply:
I can change the comment to reflect the behavior of the script. You will see at lines 835-850 I attempt to load the widget with the most recently used value. If there is no value found it defaults to create unique. Also on a reset event it sets itself to create unique. 

I could have the element always load a create unique instead of last used. Thoughts on that? That would differ from the rest of the elements as I made everything load up with last used.

Reply:
I just made a commit that makes 'create unique' the default selection at startup